sql/core/src/main/scala/org/apache/spark/sql/execution/BufferedRowIterator.java

```diff
@@ -35,6 +35,8 @@
   // used when there is no column in output
   protected UnsafeRow unsafeRow = new UnsafeRow(0);
 
+  protected boolean stopEarly = false;
```

Comment:
Since `stopEarly` is only accessed generated functions, we don't need this anymore.